website/versioned_docs/version-0.63/native-modules-android.md

```diff
@@ -1,116 +1,127 @@
 
-If you plan to make changes in Java code, we recommend enabling [Gradle Daemon](https://docs.gradle.org/2.9/userguide/gradle_daemon.html) to speed up builds.
+<figure>
+  <img src="/docs/assets/native-modules-android-open-project.png" width="500" alt="Image of opening up an Android project within a React Native app inside of Android Studio." />
+  <figcaption>Image of where you can find your Android project</figcaption>
+</figure>
 
-## The Toast Module
+We recommend using Android Studio to write your native code. Android studio is built for Android development and using it will help you resolve minor errors like code syntax quickly.
 
-This guide will use the [Toast](http://developer.android.com/reference/android/widget/Toast.html) example. Let's say we would like to be able to create a toast message from JavaScript.
+We also recommend enabling [Gradle Daemon](https://docs.gradle.org/2.9/userguide/gradle_daemon.html) to speed up builds as you iterate on Java code.
 
-We start by creating a native module. A native module is a Java class that usually extends the `ReactContextBaseJavaModule` class and implements the functionality required by the JavaScript. Our goal here is to be able to write `ToastExample.show('Awesome', ToastExample.SHORT);` from JavaScript to display a short toast on the screen.
+### Create A Custom Native Module File
 
-Create a new Java Class named `ToastModule.java` inside `android/app/src/main/java/com/your-app-name/` folder with the content below:
+The first step is to create a main custom native module file. Create a new Java class named CalendarModule.java inside `android/app/src/main/java/com/your-app-name/ folder`.
 
-```java
-// ToastModule.java
-
-package com.your-app-name;
+<figure>
+  <img src="/docs/assets/native-modules-android-add-class.png" width="700" alt="Image of adding a class called CalendarModule.java within the Android Studio." />
+  <figcaption>Image of how to add the CalendarModuleClass</figcaption>
+</figure>
 
-import android.widget.Toast;
+Then add the following content:
 
+```jsx
+package com.your-app-name; // replace com.your-app-name with your app’s name
 import com.facebook.react.bridge.NativeModule;
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.bridge.ReactContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
 import com.facebook.react.bridge.ReactMethod;
-
 import java.util.Map;
 import java.util.HashMap;
 
-public class ToastModule extends ReactContextBaseJavaModule {
-  private static ReactApplicationContext reactContext;
+public class CalendarModule extends ReactContextBaseJavaModule {
+   CalendarModule(ReactApplicationContext context) {
+       super(context);
+   }
+}
+```
+
+As you can see, your `CalendarModule` class extends the `ReactContextBaseJavaModule`. For Android, most native modules consist of a class that extends `ReactContextBaseJavaModule` and implements the functionality required by Javascript.
```

Comment:
The word "consist", how it's used here, is a bit confusing: "For Android, most native modules consist of a class". 

```suggestion
As you can see, your `CalendarModule` class extends the `ReactContextBaseJavaModule` class. For Android, Java native modules are written as classes that extend `ReactContextBaseJavaModule` and implement the functionality required by Javascript.
```